docs/user/03-70-delete-bindings-and-instances.md

```diff
@@ -4,18 +4,11 @@ Delete service bindings and service instances using Kyma dashboard or kubectl.
 
 ## Context
 
-You can only delete service instances or service bindings created in Kyma using Kyma dashboard or kubectl. You can't perform these operations using the SAP BTP cockpit.
+You can only delete service instances or service bindings created in Kyma using Kyma dashboard or kubectl. You can't perform these operations using the SAP BTP cockpit. To successfully delete a service instance, first delete its service bindings.
```

Comment:
do we need the word "successfully" here?

Reply:
Nope :)